README.md

```diff
@@ -1,4 +1,5 @@
 [![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-a873d1.svg)](https://houndci.com)
+[![Coverage Status](https://coveralls.io/repos/github/andela/heimdal-ah-backend/badge.svg?branch=master)](https://coveralls.io/github/andela/heimdal-ah-backend?branch=master)
 Authors Haven - A Social platform for the creative at heart.
```

Comment:
Add a new line before the `Authors Haven - A Social platform for the creative at heart.`

Reply:
> Add a new line before the `Authors Haven - A Social platform for the creative at heart.`

@Chidi3b3r3 feedback noted.